python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -284,25 +349,27 @@ def prepareDcrInputs(self, templateCoadd, tempExpRefList, weightList):
                                       " to the Mapper class in your obs package.")
         tempExpName = self.getTempExpDatasetName(self.warpType)
         dcrShifts = []
-        airmassList = {}
-        angleList = {}
+        airmassDict = {}
+        angleDict = {}
         for visitNum, tempExpRef in enumerate(tempExpRefList):
             visitInfo = tempExpRef.get(tempExpName + "_visitInfo")
+            visit = tempExpRef.dataId["visit"]
             airmass = visitInfo.getBoresightAirmass()
             parallacticAngle = visitInfo.getBoresightParAngle().asDegrees()
-            airmassList[tempExpRef.dataId["visit"]] = airmass
-            angleList[tempExpRef.dataId["visit"]] = parallacticAngle
+            airmassDict[visit] = airmass
+            angleDict[visit] = parallacticAngle
             if self.config.doAirmassWeight:
                 weightList[visitNum] *= airmass
             dcrShifts.append(np.max(np.abs(calculateDcr(visitInfo, templateCoadd.getWcs(),
                                                         filterInfo, self.config.dcrNumSubfilters))))
-        self.log.info("Selected airmasses:\n%s", airmassList)
-        self.log.info("Selected parallactic angles:\n%s", angleList)
+        self.log.info("Selected airmasses:\n%s", airmassDict)
+        self.log.info("Selected parallactic angles:\n%s", angleDict)
```

Comment:
How many things end up in these dicts? I fear it might be a lot of output. Maybe min/max/mean instead in `log.info`, and the full dicts in `log.debug`?

Reply:
There will be one entry for each input visit. I think that's okay, since we already log the weight and number of positive/negative sources detected in each visit.

Reply:
We talked about this on slack: what we really want would be a database with all of this information for the run. The logs are not a good place for it, but we don't have an obviously better one right now. :crying_cat_face: